examples/getting_started/example_simulation.py

```diff
@@ -23,9 +23,20 @@
     )
 
     # Simulation the Initial Guess
+    # Interpolation: Constant
     X = InitialGuess([0, 0, 0, 0])
     U = InitialGuess([-1, 1])
 
+    sol_from_initial_guess = Solution(ocp, [X, U])
+    s = sol_from_initial_guess.integrate(shooting_type=Shooting.SINGLE_CONTINUOUS)
+    print(f"Final position of q from single shooting of initial guess = {s.states['q'][:, -1]}")
+
+    # Interpolation: Each frame
+    X = np.random.rand(4, 11)
```

Comment:
This does not make much sense to create something else than a constant for X, since Shooting.SINGLE is used. It makes sense for U though...

Reply:
Yes, but if the values come from a previous optimization, X is not constant. So X created by a random function illustrates this case.